tensorboard/webapp/widgets/line_chart_v2/sub_view/line_chart_axis_view.ng.html

```diff
@@ -43,14 +47,19 @@
       <mat-icon svgIcon="edit_24px"></mat-icon>
     </button>
   </div>
-  <svg *ngIf="shouldShowMajorTicks()" class="major ticks">
-    <g *ngFor="let tick of getMajorTicks()">
-      <text [attr.x]="textXPosition(tick)" [attr.y]="textYPosition(tick)">
-        {{ getFormatter().formatShort(tick) }}
-      </text>
-      <title>{{ getFormatter().formatReadable(tick) }}</title>
-    </g>
-  </svg>
+  <div *ngIf="majorTicks.length" class="major ticks">
+    <span
+      *ngFor="let tick of majorTicks; index as i; last as isLast; trackBy: trackByMajorTick"
+      [class.major-label]="true"
```

Comment:
if this is always true, why don't we just use `class="major-label"`?

Reply:
The next line defines `class.last` and I am afraid that Angular does not exactly handle `class="foo"` and `[class.bar]="something"` correctly.

I am leaning towards keeping as is.